packages/providers/src/ports/CollectionsApiProvider/types/UpdateOrCreateItem.ts

```diff
@@ -0,0 +1,4 @@
+export interface UpdateOrCreateItem {
```

Comment:
I prefer to call it create before update. As is more natural how the actions are done: first they are created, then they are updated.